CONTRIBUTING.md

```diff
@@ -10,45 +10,81 @@ it can be incorporated into the master branch.
 To help developing `mne-bids`, you will need a few adjustments to your
 installation as shown below.
 
-##### Running tests
+Running tests
+-------------
 
-To run the tests using `pytest`, you need to have the git cloned mne-python
-repository with a local pip-install instead of the mne-python package from
-pypi. Update your installation as follows.
+### (Optional) Install development version of MNE-Python
+If you want to run the tests with a development version of MNE-Python,
+you need to clone the `mne-python` repository and install it in "editable" mode
+locally.
 
+    $ pip uninstall mne  # First, uninstall PyPI-based mne-python
     $ git clone https://github.com/mne-tools/mne-python --depth 1
-    $ cd mne-python
-    $ pip uninstall mne  # uninstall pypi mne
-    $ pip install -e .  # use the cloned repo for a local install of mne
+    $ pip install -e ./mne-python  # install from the cloned repository
```

Comment:
if you don't have a proper clone I would suggest:

`pip install -U https://github.com/mne-tools/mne-python/archive/master.zip`